examples/flows/transfer-mv-yaml/definition.yaml

```diff
@@ -8,7 +8,7 @@ States:
       Run the initial transfer operation from the source ep/source path to the
       destination ep/destination path
     Type: Action
-    ActionUrl: 'https://actions.globus.org/transfer/transfer'
+    ActionUrl: 'https://prod.actions.automate.globus.org/transfer/transfer'
```

Comment:
I don't know how complicated this would be but I think keeping our service URLs short and env agnostic would be good UX. As far as our users are concerned, they should only ever need prod and a URL with prod in it is unnecessary info. Maybe we can talk about what we'd need to do to keep using the `https://actions.globus.org/transfer/transfer` style urls

Reply:
Hmm, I don't know where this change came from. I agree that it shouldn't be here, and bad on me for not seeing it in the commit diffs.